velero/schedule/common-service-db/cs-db-br-script-cm.yaml

```diff
@@ -146,6 +152,31 @@ data:
         fi
     }
 
+    function wait_for_auth_service {
+        info "Waiting for pod platform-auth-service to restart in namespace $CSDB_NAMESPACE."  
+        deployment_exists=$(oc get deployment platform-auth-service -n $CSDB_NAMESPACE --no-headers || echo "fail")    
+        if [[ $deployment_exists != "fail" ]]; then
+            replicas=$(oc get deployment platform-auth-service -n $CSDB_NAMESPACE --no-headers -o jsonpath='{.spec.replicas}')
+            ready_replicas=$(oc get deployment platform-auth-service -n $CSDB_NAMESPACE --no-headers -o jsonpath='{.status.readyReplicas}')
+            retry_count=60
+            while [[ "$replicas" != "$ready_replicas" ]] && [[ $retry_count > 0 ]]
+            do
+                info "Wait for pod platform-auth-service to complete. Try again in 5s."
+                sleep 5
+                completed=$(oc get job $job_name -n $CSDB_NAMESPACE --no-headers | awk '{print $2}')
+                retry_count=$((retry_count-1))
+            done
```

Comment:
I had trouble getting it to complete successfully until I added:
```
replicas=$(oc get deployment platform-auth-service -n $CSDB_NAMESPACE --no-headers -o jsonpath='{.spec.replicas}')
ready_replicas=$(oc get deployment platform-auth-service -n $CSDB_NAMESPACE --no-headers -o jsonpath='{.status.readyReplicas}')
```
into the while loop like we talked about on Tuesday. After I added those lines it completed without issue.